src/MetadataUtility/Audio/Wave.cs

```diff
@@ -297,11 +307,6 @@ public static Fin<RangeHelper.Range> ScanForChunk(Stream stream, RangeHelper.Ran
             const int ChunkIdLength = 4;
             const int ChunkLengthLength = 4;
 
-            if (container.End > stream.Length)
-            {
-                throw new ArgumentOutOfRangeException(nameof(container), "container.End must be less than or equal to stream.Length");
-            }
-
```

Comment:
Removed this bit which is basically checking the outer container isn't too long, this would fail for both the format chunk and data chunk since the outer container is determined by the RIFF chunk (which we determined is also long by 44 bytes).

Reply:
The more I thought about it, do we really care if the chunk size from RIFF is too long. so long as the inner sub chunks are accurate (or practically for our purposes, so long as the format chunk is accurate)? Let me know what you think.